include/boost/uuid/random_generator.hpp

```diff
@@ -31,19 +55,21 @@ class basic_random_generator {
 public:
     typedef uuid result_type;
 
-    // default constructor creates the random number generator
+    // default constructor creates the random number generator and
+    // if the UniformRandomNumberGenerator is a PseudoRandomNumberGenerator
+    // then it gets seeded by a random_provider.
     basic_random_generator()
         : pURNG(new UniformRandomNumberGenerator)
         , generator
           ( pURNG.get()
           , distribution_type
-            ( (std::numeric_limits<unsigned long>::min)()
-            , (std::numeric_limits<unsigned long>::max)()
+            ( (std::numeric_limits<unsigned int>::min)()
```

Comment:
Not clear why these changes from ulong to uint are needed. Would probably be better to keep the original to cut down on the delta unless there's a good reason to switch.